engine/src/main/java/org/terasology/engine/core/subsystem/headless/assets/HeadlessMesh.java

```diff
@@ -32,8 +34,13 @@ public AABBfc getAABB() {
     }
 
     @Override
-    public TFloatList getVertices() {
-        return data.getVertices();
+    public VertexAttributeBinding<Vector3fc, Vector3f> vertices() {
+        return data.positions();
+    }
+
+    @Override
+    public int elements() {
```

Comment:
I don't think `elements` is a good name for something that returns a number, but maybe this is the right name for this domain 🤷  I'd prefer something like `elementCount` to emphasize that this is a number, opposed to `elements` which, to me, suggests that this method returns the elements of this mesh (as in, vertices, edges, or whatever the "elements" of a mesh are).